### PR TITLE
Add more functional classes

### PR DESCRIPTION
**Pull request by marcofugaro**
Taken from all the latest internal projects which use Tachyons

### CODE REVIEW

File: tachyons-extra.css [hunk 1]
```diff
@@ -83,3 +133,48 @@
 .prewrap {
   white-space: pre-wrap;
 }
+
+.user-select-none, .user-select-none * {
+  user-select: none;
+}
+
+.transition {
```
Comment (lucafalasco) [inline tachyons-extra.css:141]: we could also think of more specific transitions and/or different timing

**Review by lucafalasco — APPROVED** [review]
👍

File: tachyons-extra.css (hunk 1 above)
Comment (marcofugaro) [reply tachyons-extra.css:141]: `.transition--fast` and `.transition--slow`?

Comment (lucafalasco) [reply tachyons-extra.css:141]: I was thinking something like `.transition-all-1` `.transition-all-2` and `.transition-width-1` `.transition-width-2` etc.. which is more "tachyons style" but maybe it also makes sense to keep those more explicit in this case. 
Anyway I think it's important to differentiate the type of transition too (maybe we only need to animate a specific css property and no others)

Comment (marcofugaro) [reply tachyons-extra.css:141]: I don't think specific transitions for every property should be in `tachyons-extra`, just create the transition with the property you need when you need it. The `transition` is really for testing, plug and play. You shouldn't animate the width property anyway.

Comment (lucafalasco) [reply tachyons-extra.css:141]: 🤔not really sure why animating the width property should be avoided. 
If the `.transition` class is just for testing purposes then I don't think it makes sense to include it in the bundle, you can just momentarily put it as an inline style attribute for testing. 
I think having a predefined set of transition classes for most used properties can actually be worth it. And then of course you can always create a custom one if you need it.
Curious what are @accurat/developers thoughts about this.

Comment (lucafalasco) [reply tachyons-extra.css:141]: Better discuss this here anyway: https://github.com/accurat/tachyons-extra/issues/4

**Comment by caesarsol** [comment]
Nice!! 👍

**Pull request merged**